app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __del__(self) -> None:
+        Animal.alive.remove(self)
+
+    def __repr__(self) -> None:
```

Comment:
The `__repr__` method should return a string, not `None`. Change the return type annotation to `str`.